spring-cloud-consul-core/src/main/java/org/springframework/cloud/consul/ConsulHealthIndicatorProperties.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2013-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.consul;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.core.style.ToStringCreator;
+
+/**
+ * Configuration properties for {@link ConsulHealthIndicator}.
+ *
+ * @author Chris Bono
+ */
+@ConfigurationProperties("spring.cloud.consul.health-indicator")
+public class ConsulHealthIndicatorProperties {
+
+	private boolean enabled = true;
+
+	private boolean includeServicesQuery = true;
```

Comment:
If this was targeted to the 2.2.x branch, we could do 'true' there and then in master, since it is a major release, change to 'false'.

Reply:
Good idea. I will adjust.

Reply:
If this is a new property in master only, could we not just make the default `false` there and not put in `2.2.x`?

Reply:
I went ahead and targeted 2.2.x in https://github.com/spring-cloud/spring-cloud-consul/pull/653 . I believe this can be closed in favor of that.